lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -139,6 +137,20 @@ class LineChartPainter extends AxisChartPainter<LineChartData>
     }
   }
 
+  void clipToBorder(ui.Canvas canvas, ui.Size size,) {
+    final double halfStrokeWidth = clearAroundBorderPaint.strokeWidth / 2;
+    final Rect rect = Rect.fromLTRB(
+      getLeftOffsetDrawSize() - halfStrokeWidth,
+      getTopOffsetDrawSize() - halfStrokeWidth,
+      size.width -
+          (getExtraNeededHorizontalSpace() - getLeftOffsetDrawSize()) +
+          halfStrokeWidth,
+      size.height- (getExtraNeededVerticalSpace() - getTopOffsetDrawSize()) + halfStrokeWidth,
+    );
+
+    canvas.clipRect(rect, clipOp: ClipOp.intersect);
```

Comment:
The default value of `clipOp` is `ClipOp.intersect`, then you don't have to to define it.

Reply:
Fixed